apache/dolphinscheduler
# [Feature-13481][Resource Center] Add support for Azure Blob Storage in the resource center

## Purpose of the pull request
* close: #13481 



## Brief change log

* Add support for Azure Blob Storage in the resource center

## Verify this pull request

UT and manually tested

1. upload files
<img width="1503" alt="截屏2023-05-11 11 23 16" src="https://github.com/apache/dolphinscheduler/assets/38122586/cac9448b-2fff-47c9-b3cf-6831680e4672">

2. check files on the Azure
<img width="919" alt="截屏2023-05-11 11 32 24" src="https://github.com/apache/dolphinscheduler/assets/38122586/5947222c-4aec-4162-a2df-fd1f089f27f1">

* Note that there is a duplicate empty file `<no name>` since Azure blob storage does not support having empty folders. But this does not affect the file display in the ds resource center
